### PR TITLE
Add support for double quotes in samplesheets

### PR DESCRIPTION
**Pull request by nvnieuwk**
cf. https://nfcore.slack.com/archives/C056RQB10LU/p1700559108707619

### CODE REVIEW

File: plugins/nf-validation/src/main/nextflow/validation/SamplesheetConverter.groovy [hunk 1]
```diff
@@ -80,7 +80,7 @@ class SamplesheetConverter {
         }
         else {
             Path fileSamplesheet = Nextflow.file(samplesheetFile) as Path
-            samplesheetList = fileSamplesheet.splitCsv(header:containsHeader ?: ["empty"], strip:true, sep:delimiter, quote:'"')
+            samplesheetList = fileSamplesheet.splitCsv(header:containsHeader ?: ["empty"], strip:true, sep:delimiter, quote:'\"')
```
Comment (maxulysse) [inline plugins/nf-validation/src/main/nextflow/validation/SamplesheetConverter.groovy:83]: oh so that wasn't working properly before?

**Review by mirpedrol — APPROVED** [review]
LGTM!

File: plugins/nf-validation/src/main/nextflow/validation/SamplesheetConverter.groovy (hunk 1 above)
Comment (nvnieuwk) [reply plugins/nf-validation/src/main/nextflow/validation/SamplesheetConverter.groovy:83]: Yes that was working :), this part was for the samplesheet conversion but somehow we didn't add it for the validation :grin:

Comment (maxulysse) [reply plugins/nf-validation/src/main/nextflow/validation/SamplesheetConverter.groovy:83]: Oh I see

**Pull request merged**